common/src/main/java/com/bakdata/dedupe/fusion/ConflictResolutionFusion.java

```diff
@@ -68,7 +68,8 @@ public class ConflictResolutionFusion<R> implements Fusion<R> {
     @NonNull ConflictResolution<R, R> rootResolution;
 
     @Getter(value = AccessLevel.PRIVATE, lazy = true)
-    @NonNull Map<@NonNull String, @NonNull Source> sourceByName =
+    @NonNull
+    Map<String, Source> sourceByName =
```

Comment:
Annotations on generics caused Jacoco to fail

Reply:
Maybe we should exclude code generated by lombok? It seems like Jacoco respects the `@Generated` so we could try adding a lombok.config with `lombok.addLombokGeneratedAnnotation = true`

Reply:
It is there https://github.com/bakdata/dedupe/blob/master/common/lombok.config#L3